scalalib/worker/src/ZincWorkerImpl.scala

```diff
@@ -21,6 +22,45 @@ case class MockedLookup(am: File => Optional[CompileAnalysis]) extends PerClassp
     Locate.definesClass(classpathEntry)
 }
 
+class ZincProblem(base: xsbti.Problem) extends Problem {
```

Comment:
Those are dumb decorators/adapters to avoid the zinc dependencies in the core modules